### PR TITLE
Update Atlassian Confluence Readme

### PR DESCRIPTION
**Pull request by ellis-elastic**
Added link to Atlassian documentation

Type of change

- Enhancement

## What does this PR do?

Added link to Atlassian documentation

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

**Review by r00tu53r — COMMENTED** [review]
The deletions indicated in `docs/README.md` for field docs (`host.os.name.text ..... user.target.name.text`) shouldn't be here. 

I think its because the updated `docs/README.md` has not been committed/pushed. Please apply these steps for all the open PRs.

- Change readme under `_dev/build/docs/README.md` (if it exists)
- Run `elastic-package build`
- Commit `_dev/build/docs/README.md` and `docs/README.md`
- Push changes

File: packages/atlassian_confluence/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.1.2"
+  changes:
+    - description: Update readme
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/3063
```
[the remark this answers is not in the document]
Comment (ellis-elastic) [reply packages/atlassian_confluence/changelog.yml]: Done

**Comment by ellis-elastic** [comment]
I've installed v0.46.1 and re run build

**Review by efd6 — APPROVED** [review]

**Pull request merged**